packages/sanity/src/core/studio/components/navbar/resources/helper-functions/helpResources.ts

```diff
@@ -19,7 +19,7 @@ export function getHelpResources(
       Builds to: `{m: ['sanity@3.0.2']}' and serializes to: `?m=sanity@3.0.2`.
       Final format will be e.g. https://api.sanity.io/v1/help?tag=sanity.studio.module.version-check&m=sanity%403.0.2
       */
-    query: {m: [`sanity@${SANITY_VERSION}`]},
+    query: {m: [`sanity@${SANITY_VERSION}`], locale},
```

Comment:
can the above todo comment now be removed?

Reply:
Oh, thanks - yes! - missed it 😅